src/edu/gatech/cs2340/trydent/TrydentEngine.java

```diff
@@ -0,0 +1,145 @@
+package edu.gatech.cs2340.trydent;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Main class that kicks off the TrydentEngine.
+ * 
+ * As a developer using the TrydentEngine, all you have to do is call
+ * TrydentEngine.start() to get started.
```

Comment:
It's pretty antiquated but I could see an argument that it's the most easily understandable methods of starting up the library. I think doing some static initializer block stuff might just be doing it differently for the sake of doing it differently and isn't really any better.